toolchain/check/handle_binding_pattern.cpp

```diff
@@ -115,6 +115,10 @@ auto HandleAnyBindingPattern(Context& context, Parse::NodeId parse_node,
       if (context_parse_node_kind == Parse::NodeKind::ReturnedModifier) {
         RegisterReturnedVar(context, bind_id);
       }
+      if (auto current_scope_id = context.current_scope_id();
+          current_scope_id == SemIR::NameScopeId::Package) {
```

Comment:
The named variable doesn't seem necessary here. Would this work instead?

```suggestion
      if (context.current_scope_id() == SemIR::NameScopeId::Package) {
```

Reply:
Definitely! This totally has slipped my attention.
I updated them.